luatest/replica_set.lua

```diff
@@ -183,12 +183,12 @@ function ReplicaSet:stop()
     end
 end
 
---- Stop all servers in the replica set and clean their working directories.
+--- Stop all servers in the replica set and save its artifacts.
+-- Use this method at the end of test suite.
```

Comment:
This function should be used only at the end of the test (`after_test`, `after_each`, `after_all` hooks) to terminate all server processes in the replica set. Besides process termination, it saves the contents of each server working directory to the `<vardir>/artifacts` directory for further analysis if the test fails.

Reply:
Fixed